features/verify_helper.py

```diff
@@ -16,3 +21,60 @@ async def has_role(self, user, role_name: str) -> bool:
             guild = await self.bot.fetch_guild(config.guild_id)
             member = await guild.fetch_member(user.id)
             return utils.has_role(member, role_name)
+
+    def get_user_details(self, id: str) -> Optional[dict]:
+        headers = {"Authorization": f"Bearer {config.vut_api_key}"}
+        url = "https://www.vut.cz/api/"
+        res = requests.get(f"{url}person/v1/{id}/pusobeni-osoby", headers=headers)
+        if res.status_code != 200:
+            if res.status_code in [401, 403]:
+                raise Exception("Invalid API key")
+            elif res.status_code == 429:
+                raise Exception("Rate limit exceeded")
+            # Login not found
+            return None
+        return res.json()
+
+    def _parse_relation(self, user: dict) -> Optional[str]:
+        """Parse user relations and return year, programee and faculty for students,
+        `employee` for FIT employees, None for others."""
+        ret = None  # rule out students that are also employees or have multiple studies
+        for relation in user['vztahy']:
+            # student
+            if 'rok_studia' in relation.keys():
+                ret = f"{relation['fakulta']['zkratka']} {relation['obor']['zkratka']} " \
+                      f"{relation['rok_studia']}"
+                # do not return yet if not FIT, check for all relations if student has multiple studies
+                if relation['fakulta']['zkratka'] == "FIT":
+                    return ret
+            elif 'ustav' in relation.keys() and relation['ustav']['fakulta']['zkratka'] == "FIT":
+                # FIT employee, replace only if not student
+                ret = 'employee' if not ret else ret
```

Comment:
```suggestion
                ret = ret or 'employee'
```
if you want to be fancy I guess